packages/plugins/local-storage/src/local-database.ts

```diff
@@ -201,19 +201,28 @@ class LocalDatabase extends pluginUtils.Plugin<{}> implements Storage {
     const packagePath: string = this._getLocalStoragePath(
       packageAccess ? packageAccess.storage : undefined
     );
-    debug('storage path selected: ', packagePath);
+    debug('storage path selected %o', packagePath);
     if (_.isString(packagePath) === false) {
       debug('the package %o has no storage defined ', packageName);
       throw errorUtils.getInternalError('storage not found or implemented');
     }
 
+    const storagePath = path.isAbsolute(packagePath) ? packagePath : this.getStoragePath();
```

Comment:
I don't understand this change, maybe you could give me some  context, the method `_getLocalStoragePath()` ->  calls -> `this.getStoragePath();` and already check inside if is Absolute and return that path.

Note: I'd need to check the `this.config.config_path` line, it is indeed deprecated https://github.com/verdaccio/verdaccio/blob/fdf44a66c99733645783e326d5125b186aec96ff/packages/config/src/parse.ts#L18 but I wonder why still here.

Reply:
I fixed the storage path. 

afaict that was the only place in the plugin still using `config_path`.  `packages/config/src/config.ts` takes care of mapping it to `configPath` if necessary.